skills/xavier-cognitive-memory/SKILL.md

```
@@ -0,0 +1,162 @@
+# Escaneo de Code-Graph
+curl -s -X POST http://localhost:8006/v1/code/scan \
+  -H "Authorization: Bearer $XAVIER_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{"path": "."}'
+```
+
+---
+
+## 3. Protocolo Estándar de Ejecución para Agentes
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)